roles/openshift_prometheus_operator/files/assets/alertmanager/alertmanager.yaml

```diff
@@ -11,7 +11,12 @@ route:
       alertname: DeadMansSwitch
     receiver: 'null'
   - match:
-      alertname: K8SControllerManagerDown
+      alertname: OpenShiftApiserverDown
+      severity: critical
+    receiver: 'null'
```

Comment:
I think this should be `autoheal` so that the alert will be sent to the auto-heal service.

Reply:
Agree- I'll let you change that once you've tested it all working together